gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -707,7 +707,8 @@ private void parseInputQuery(String query) {
   protected JdbcProvider createJdbcSource() {
     String driver = this.workUnit.getProp(ConfigurationKeys.SOURCE_CONN_DRIVER);
     String userName = this.workUnit.getProp(ConfigurationKeys.SOURCE_CONN_USERNAME);
-    String password = this.workUnit.getProp(ConfigurationKeys.SOURCE_CONN_PASSWORD);
+    String password =
+        PasswordUtils.decryptPassword(this.workUnit, this.workUnit.getProp(ConfigurationKeys.SOURCE_CONN_PASSWORD));
```

Comment:
We still need to support the case where the password are un-encrypted fo backwards compatibility.  We probably need a readPassword() method in PasswordUtils that will check if the password matches ENC(.*) than it should be decrypted, otherwise return the plain text.